target/linux/brcmiproc/image/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2016-2017 Yousong Zhou <yszhou4tech@gmail.com>
+# Copyright (C) 2018 Jianhui Zhao <jianhuizhao329@gmail.com>
+# Copyright (C) 2019 Samik Gupta <samik.gupta@broadcom.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+DTS_DIR := $(DTS_DIR)/broadcom
+
+define Image/InstallKernel
+	$(foreach k,$(filter Image,$(KERNELNAME)), \
+		cp $(KDIR)/$(KERNELNAME) $(BIN_DIR)/$(IMG_PREFIX)$(if $(PROFILE_SANITIZED),-$(PROFILE_SANITIZED))-$(k) \
+	)
+	$$(foreach d,$$(shell ls $$(KDIR)/image-*.dtb),cp $$(d) $$(patsubst $$(KDIR)/image-%,$$(BIN_DIR)/$$(IMG_PREFIX)$$(if $$(PROFILE_SANITIZED),-$$(PROFILE_SANITIZED))-%,$$(d));)
+endef
+
+define Image/Build
+	$(call Image/Build/$(1))
+	dd if=$(KDIR)/root.$(1) bs=128k conv=sync | \
+		gzip -9n >$(BIN_DIR)/$(IMG_PREFIX)$(if $(PROFILE_SANITIZED),-$(PROFILE_SANITIZED))-rootfs.$(1).gz
+endef
+
+ifeq ($(SUBTARGET),bcm58800)
+define Device/broadcom_ps225-hxx
+	DEVICE_VENDOR := Broadcom
+	DEVICE_MODEL := PS225-HXX
```

Comment:
In the kernel DTS they call this device "Stingray PS225xx". When googling, I find mostly just "PS225", with and without the Stingray prefix. Can you shed light on what would be the correct name?

Despite, we have an option to include alternative names, which in this case would be interesting for the board name:
```
DEVICE_ALT0_VENDOR := Broadcom
DEVICE_ALT0_MODEL := BCM958802A802x
```

Reply:
Stingray is the trade name of the BCM58800 chip. PS225 is a board based on that chip. The link below (please go to the bottom of the page) might be helpful because it shows the chip and boards (made by Broadcom) clearly.  https://www.broadcom.com/products/ethernet-connectivity/smartnic